keps/sig-scheduling/1923-prefer-nominated-node/README.md

```diff
@@ -146,10 +143,170 @@ _This section must be completed when targeting alpha to a release._
     - Feature gate name: PreferNominatedNode
     - Components depending on the feature gate: kube-scheduler
 
+* **Does enabling the feature change any default behavior?**
+  Yes. The pod with the nominated node has been set will be evaluated first in any scheduling cycle,
+  this is only the default process logic that is handled by scheduler, end-user will not
+  and need not aware of any difference.
+
+* **Can the feature be disabled once it has been enabled (i.e. can we roll back
+  the enablement)?**
+  We can roll back the enablement by restarting scheduler with the feature gate turned off, otherwise
+  there is no way to disable this feature.
+
+* **What happens if we reenable the feature if it was previously rolled back?**
+  Nothing needs to be aware of for the end-user.
```

Comment:
The feature will start working again when scheduling pods.

Reply:
Done